.github/workflows/docs.yml

```diff
@@ -77,7 +77,9 @@ jobs:
       - name: Build
         run: |
           source python/venv/bin/activate
-          mkdocs build
+          var=$(echo `${{ github.event.client_payload.tag || github.ref_name }` | sed 's/^python-v//')
```

Comment:
I believe there is an error here with a missing second `}`

Reply:
Ooops, too many github accounts :)

Reply:
Typo :) Fixed it now